components/API/LayerList/index.jsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import './style.scss';
+
+class LayersList extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      layers: [],
+      message: 'Loading...',
+      default: []
+    };
+  }
+
+  componentWillMount() {
+    const { application, path, dataset } = this.props;
+    this.getLayers(path, application, dataset);
+
+    // Bindings
+    this.sortLayers = this.sortLayers.bind(this);
+  }
+
+  render() {
+    const { path } = this.props;
+
+    return (
+      <div className="c-layers-list">
+        <div className="intro">
+          <h4 className="count">{ this.state.layers.length } layers</h4>
+          <div className="sort">
+            Sort by
+            <select onChange={this.sortLayers}>
+              <option value="0">Default</option>
+              <option value="1">A - Z</option>
+              <option value="2">Z - A</option>
+            </select>
+          </div>
+        </div>
+        { this.state.layers.length ?
+          <table className="list">
+            <thead>
+              <tr>
+                <th>Name</th>
+                <th>Has Config</th>
+                <th>Has Legend</th>
+                <th></th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              {this.state.layers.map((layer, i) => {
+                return (
+                  <tr key={i} className="item">
+                    <td>{layer.name ? layer.name : layer.msg }</td>
+                    <td>{layer.config ? 'true' : 'false'}</td>
+                    <td>{layer.legend ? 'true' : 'false'}</td>
+                    <td>{layer.dataset ? <a href={`/dataset?id=${layer.dataset}`} className="btn-edit">Dataset</a> : 'Dataset' }</td>
+                    <td>{layer.id && <a href={`${path}?id=${layer.id}`} className="btn-edit">Edit</a> }</td>
```

Comment:
Be careful, you need to pass the dataset and the layer id for editing the layer. Just as we are doing with the widget list